Reference/Templating/Mvc/partial-views-v7.md

```diff
@@ -0,0 +1,142 @@
+---
+versionFrom: 7.0.0
+needsV8Update: "true"
```

Comment:
Doesn't look like this needs Umbraco 8 updating anymore 💪 

Should we get rid of it? 😁 
```suggestion

```